capstone/src/main/webapp/club-registration.html

```diff
@@ -22,14 +22,10 @@ <h2>Club Registration</h2>
         <label for="website">Website:</label>
         <input type="text" name="website" id="club-website">
       </div>      
-      <div class="form-group">
-        <label for="calendar-link">Google Calendar link:</label>
-        <input type="text" name="calendar-link" id="club-calendar-link">
-      </div>
-      <div class="form-group">
+      <!-- <div class="form-group">
```

Comment:
nit: remove commented code. If you need to access it later on, I think it would be better to look in the history instead of commenting it